ripple/target/appengine-staging/__static__/auth.js

```diff
@@ -0,0 +1,133 @@
+// function to sign up with Google
+function signUpWithGoogle() {
+  var provider = new firebase.auth.GoogleAuthProvider();
+  provider.addScope('profile');
+  auth.signInWithPopup(provider).then(function(result) {
+    // window.location = 'landingbusiness.html';
+    console.log("Success: Google account linked");
+    var token = result.credential.accessToken;
+    var user = result.user;
+    var name = user.displayName;
+    var uid = user.uid;
+    var email = user.email;
+    var isbusinessOwner = false;
+    console.log("name: " + name);
+    console.log("uid: " + uid);
+    sendEmailVerification(uid);
+    addNewUser(uid, name, isBusinessOwner, email);
+  }).catch(function(error) {
+    console.log(error);
+    // Handle Errors here.
+    var errorCode = error.code;
+    var errorMessage = error.message;
+    var email = error.email;
+    var credential = error.credential;
+    // ...
+  });
+}
+
+function addNewUser(uid, name, isBusinessOwner, email) {
+  console.log("addNewUser("+ uid + ", " + name + ", " + isBusinessOwner + ")");
+  db.collection("users").doc(uid).set({
+    uid: uid,
+    name: name,
+    isBusinessOwner: isBusinessOwner,
+    email: email,
+    // blobKey for default avatar photo
+    blobKey: "XEkBWMfbItUDt70PromXrQ",
```

Comment:
Now that I am seeing it again, is this something that is provided and you are just adding it to github?

Reply:
I believe target directory files are automatically generated based on the code stored in my src directory. Changes in the src are detected by the target and changed accordingly.